cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/BatchFileFilter.java

```diff
@@ -52,54 +52,59 @@
 public class BatchFileFilter extends Configured implements PathFilter {
 
   private static final Logger LOG = LoggerFactory.getLogger(FileBatchSource.class);
-  //length of 'YYYY-MM-dd-HH-mm"
-  private static final int DATE_LENGTH = 16;
+  //length of 'YYYY-MM-dd-HH"
+  private static final int DATE_LENGTH = 13;
```

Comment:
this would make DATE_LENGTH non-static since sdf is not static for concurrency reasons. would we prefer that? or just eliminate the field entirely and use sdf.toPattern().length() directly.